public/app/features/canvas/elements/button.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { PureComponent } from 'react';
 
+import { PluginState } from '@grafana/data/src';
```

Comment:
```suggestion
import { PluginState } from '@grafana/data';
```

We only need the `src` path for things that are not exported